### PR TITLE
fix(combobox): add properties documentation (VIV-1142)

### PR DESCRIPTION
**Pull request by rinaok**
https://jira.vonage.com/browse/VIV-1142

https://www.fast.design/docs/components/combobox

The `selectedOptions` and other properties were not documented in the docs.

### CODE REVIEW

File: libs/components/src/lib/combobox/README.md [hunk 1]
```diff
@@ -135,6 +135,29 @@ Add the `disabled` attribute to disable the combobox.
...
+
+- Type: `Option[]`
+- Default: `[]`
+
+### selectedOptions
+
+A collection of the selected options.
+
+- Type: `Option[]`
+- Default: `[]`
```
Comment (RichardHelm) [inline libs/components/src/lib/combobox/README.md:152]: I'm assuming these are read-only? If so we should probably mention it

```suggestion
### options

A read-only list of options.

- Type: `Option[]`
- Default: `[]`

### selectedOptions

A read-only collection of the selected options.

- Type: `Option[]`
- Default: `[]`
```

File: libs/components/src/lib/combobox/README.md [hunk 2]
```diff
@@ -135,6 +135,29 @@ Add the `disabled` attribute to disable the combobox.
...
+
+### selectedIndex
+
+The index of the selected option.
+
+- Type: `number`
+- Default: `undefined`
```
Comment (RichardHelm) [inline libs/components/src/lib/combobox/README.md:159]: It's -1 when nothing is selected? So I think we should document that as well
```suggestion
### selectedIndex

The index of the selected option or -1 if no option is selected.

- Type: `number`
- Default: `undefined`
```

File: libs/components/src/lib/combobox/combobox.spec.ts [hunk 3]
```diff
@@ -94,4 +95,26 @@ describe('vwc-combobox', () => {
 				.toEqual(true);
 		});
 	});
+
+	describe('selectedIndex', () => {
```
Comment (RichardHelm) [inline libs/components/src/lib/combobox/combobox.spec.ts:99]: Maybe we should add a test for `options` as well
We should test all the things we describe in our documentation

Comment (rinaok) [reply libs/components/src/lib/combobox/combobox.spec.ts:99]: added

File: libs/components/src/lib/combobox/combobox.spec.ts [hunk 4]
```diff
@@ -94,4 +95,42 @@ describe('vwc-combobox', () => {
 				.toEqual(true);
 		});
 	});
+
```
Comment (YonatanKra) [inline libs/components/src/lib/combobox/combobox.spec.ts:98]: Missing API in tests:
`selectedOptions`

File: libs/components/src/lib/select/README.md [hunk 5]
```diff
@@ -364,6 +364,28 @@ You can specify width on the `vwc-select` if required (the list-box will not be
...
+## Properties
+
+### options
+
+List of options.
+
+- Type: `Option[]`
+- Default: `[]`
+
+### selectedOptions
```
Comment (YonatanKra) [inline libs/components/src/lib/select/README.md:376]: Not tested :)

**Review by YonatanKra — CHANGES_REQUESTED** [review]
Small tests :)

File: libs/components/src/lib/select/README.md (hunk 5 above)
Comment (rinaok) [reply libs/components/src/lib/select/README.md:376]: it is tested 

image

File: libs/components/src/lib/combobox/combobox.spec.ts (hunk 4 above)
Comment (rinaok) [reply libs/components/src/lib/combobox/combobox.spec.ts:98]: image

File: libs/components/src/lib/listbox/listbox.spec.ts [hunk 6]
```diff
@@ -106,5 +107,43 @@ describe('vwc-listbox', () => {
 			expect(element.getAttribute('aria-activedescendant')).toEqual('');
 		});
 	});
+
```
[the remark this answers is not in the document]
Comment (YonatanKra) [reply libs/components/src/lib/listbox/listbox.spec.ts:110]: Does it have its own `describe`?

File: libs/components/src/lib/select/ui.test.ts-snapshots/snapshots-select-Desktop-Firefox-linux.png
Comment (RichardHelm) [inline libs/components/src/lib/select/ui.test.ts-snapshots/snapshots-select-Desktop-Firefox-linux.png:1]: In the new screenshot one of the menu is not open anymore. This is a problem with this test where it sometimes is not open making it flaky.
We really need to fix the test, but in the meantime could you please regenerate the screenshot with the menu open?

**Review by RichardHelm — CHANGES_REQUESTED** [review]

**Comment by RichardHelm** [comment]
@rinaok Looks great, I've copied the wording form combobox to the other READMEs as well
Just the screenshot needs updating

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**